src/ide/highlight.rs

```diff
@@ -246,6 +246,7 @@ pub fn highlight(node: &LinkedNode) -> Option<Tag> {
         SyntaxKind::LoopContinue => None,
         SyntaxKind::FuncReturn => None,
         SyntaxKind::Destructuring => None,
+        SyntaxKind::DestructAssignment => None,
```

Comment:
are all assignments now this, even `a = 1`? if so, we could simply call it `Assignment` it guess and remove that binary operator. if `a = 1` isn't this right now, should it be?

Reply:
No normal assignments still use the binary operator. I tried removing moving normal assigns into the assignment, but changed it back to this. I'm not sure why anymore. I could try again and see what happens.

Reply:
It should be possible to split them up but then the implementation of `=` and `+=` would get split up. I don't see any benefits from splitting them either so I think they should just stay as they are to avoid introducing bugs.